metadata-analysis/Metadata-Analysis.Rmd

```diff
@@ -217,13 +389,160 @@ Eva did not begin tracking downloads until December 29, 2022.
+  ggplot(aes(x = MonthEnding, y = n, group = EventCategory, color = EventCategory, label = n)) +
+  geom_line() +
+  geom_point() +
+  geom_text(nudge_x = .5, color = "gray15", check_overlap = TRUE) +
+  scale_color_brewer(palette = "Dark2") +
+  scale_x_date(date_breaks = "1 month", date_labels = "%B") +
+  labs(title = "Monthly Eva Downloads - Total Downloads",
+       x = "Month Ending",
+       y = "") +
+  theme_minimal() +
+  theme(legend.position = "bottom") +
+  guides(color = guide_legend(ncol = 2))
+
+
+```
+
+
+## Demo Mode Stats
+
+```{r fig.width=10, fig.height=5}
+
+counts_by_week %>%
+  filter(str_detect(Event, "Switched to demo mode")) %>%
+  ggplot(aes(x = WeekEnding, y = n, group = Event, color = Event, label = n)) +
+  geom_line(na.rm = FALSE) +
+  geom_point(na.rm = FALSE) +
+  geom_text(nudge_x = .5, color = "gray15", check_overlap = TRUE) +
+  scale_color_brewer(palette = "Set1") +
+  scale_x_date(date_breaks = "1 week", date_labels = "%m/%d") +
+  labs(title = "Weekly Switches to Demo Mode",
+       x = "Week Ending",
+       y = "") +
+  theme_minimal() +
+  theme(legend.position = "bottom") +
+  guides(color = guide_legend(ncol = 2))
+  
+```
+
+```{r fig.width=7, fig.height=6}
+
+staging_last6months %>%
+  filter(str_detect(Event, "User on") &
+        str_detect(Event, "DEMO MODE") &
+           !(Event %in% c("User on tabHome", "User on tabHome - DEMO MODE", "User on tabUpload - DEMO MODE", "User on tabUpload", "User on"))) %>%
+  distinct(SessionID, Event, .keep_all = TRUE) %>%
+  mutate(Tab = case_when(
+    str_detect(Event, "tabChangelog") ~ "Changelog",
+    str_detect(Event, "tabClientCount") ~ "Client Counts",
+    str_detect(Event, "tabDQOrg") ~ "Organization DQ",
+    str_detect(Event, "tabDQSystem") ~ "System DQ",
+    str_detect(Event, "tabLocalSettings") ~ "Local Settings",
+    str_detect(Event, "tabPDDE") ~ "PDDE DQ",
+    TRUE ~ as.character(Event)), 
+    Tab = factor(Tab, levels = c("Local Settings", "Client Counts", "PDDE DQ", "System DQ", "Organization DQ", "Changelog"))) %>%
+  count(Tab) %>%
+  ggplot(aes(x = Tab, y = n, fill = Tab)) +
+  geom_bar(stat = "identity") +
+  geom_text(aes(label = scales::percent(n/sum(n))), color = "gray15", vjust = -0.5) +
+  scale_fill_manual(values = 
+                   c("Changelog" = "#f94144",
+                      "Client Counts" = "#f3722c",
+                      "Organization DQ" = "#f8961e",
+                      "System DQ" = "#f9c74f",
+                      "Local Settings" = "#90be6d",
+                      "PDDE DQ" = "#43aa8b")) +
+ labs(title = "Tabs Visited in the Last 6 Months",
+       x = "Tab",
+       y = "Count") +
+  theme_minimal() +
+  theme(legend.position = "bottom") +
+  guides(fill = guide_legend(ncol = 2))
+
+```
+
+```{r fig.width=10, fig.height=5}
+
+counts_by_week %>%
+  filter(str_detect(Event, "Download")) %>%
+  filter(str_detect(Event, "DEMO MODE")) %>%
```

Comment:
these filter functions back to back may be causing problems- you would generally say
`filter(str_detect(Event, "Download") | str_detect(Event, "DEMO MODE"))` 
it may be reading these like:
`filter(str_detect(Event, "Download") & str_detect(Event, "DEMO MODE"))`

Reply:
Seems to work fine now